docs/source-pytorch/fabric/fabric.rst

```diff
@@ -589,3 +613,22 @@ For single-device strategies, it is a no-op. There are strategies that don't sup
 - xla
 
 For these, the context manager falls back to a no-op and emits a warning.
+
+
+log and log_dict
```

Comment:
This addition and the one above look repeated. They basically say the same thing. Keep one? Merge them?

Reply:
The above is for the loggers Fabric argument, the one here if for the method(s). I want people to find this information regardless of where they look. This is very brief, there will be a separate guide page that goes into more details.

Reply:
Organization will mainly happen in https://github.com/Lightning-AI/lightning/pull/16291 and following PRs.